### DISCUSSION TITLE
Obfuscation script expiry of PyArmor

### DISCUSSION THREAD

**Discussion by msraman84** [dashingsoft/pyarmor#1386] [General]
Hi,
If a python script is obfuscated with trial version of pyarmor, can we use the obfuscated script permanently without any expiry date? Please clarify.

**Comment by jondy** [comment]
Yes. Pyarmor has nothing control on obfuscated scripts.

**Reply by msraman84** [reply]
Thanks Jondy. Can you share email id for interacting with the product dev and sales team?

**Reply by msraman84** [reply]
I came across pyarmor@163.com ID. However, I am unable to get response for this ID.

**Comment by jondy** [comment]
@msraman84 pyarmor@163.com only handles business questions, please ask technical question here.
